frontend/comfywatchman-dashboard/src/index.css

```
@@ -0,0 +1,3660 @@
+.cw-job-row.is-planned svg {
+  color: #f59e0b;
+}
+
+.cw-detail {
+  display: flex;
+  flex-direction: column;
+  gap: 16px;
+}
+
+.cw-pill {
+  border: 1px solid currentColor;
```

Stylelint (17.11.1)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)